From 6b2c7479d6d33249c2385d8a36d8ee9e61e1a519 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Sat, 19 Nov 2016 09:45:45 +0200
Subject: [PATCH] Configure: also zero out high bytes of 80-bit ldnan

These are currently zero anyway, but things are probably not guaranteed
to stay so.
---
 Configure | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Configure ./Configure
index f561a87e19..845fc43d76 100755
--- a/Configure
+++ ./Configure
@@ -20702,6 +20702,7 @@ int main(int argc, char *argv[]) {
 # if LONG_DOUBLEKIND == 3 || LONG_DOUBLEKIND == 4
 /* the 80-bit long doubles might have garbage in their excess bytes */
     memset((char *)&ldinf + 10, '\0', LONG_DOUBLESIZE - 10);
+    memset((char *)&ldnan + 10, '\0', LONG_DOUBLESIZE - 10);
 # endif
   if (argc == 2) {
     switch (argv[1][0]) {
-- 
2.12.2

